<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventEditorForm</class>
 <widget class="QDialog" name="EventEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Event relation information</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="eventRoleLabel">
        <property name="text">
         <string>Role</string>
        </property>
        <property name="buddy">
         <cstring>eventRoleComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="eventRoleComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Event information</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="eventTypeLabel">
        <property name="text">
         <string>Event type</string>
        </property>
        <property name="buddy">
         <cstring>eventTypeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="eventTypeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="eventDateLabel">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
        <item>
         <widget class="QLineEdit" name="eventDatePicker"/>
        </item>
        <item>
         <widget class="QPushButton" name="eventDateEditButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-entry"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="eventNameLabel">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>eventNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="eventNameEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout2" stretch="0,0">
        <item>
         <widget class="KRichTextEdit" name="noteEdit"/>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QPushButton" name="noteEditButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-entry"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KRichTextEdit</class>
   <extends>QTextEdit</extends>
   <header>KRichTextEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtons</sender>
   <signal>accepted()</signal>
   <receiver>EventEditorForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>EventEditorForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
